test/cql-pytest/test_describe.py

```diff
@@ -1372,52 +1373,54 @@ def get_create_stmt(self) -> str:
 
 ###
 
-def test_create_role_with_salted_hash(cql):
+def test_create_role_with_hashed_password(cql):
     """
-    Verify that creating a role with a salted hash works correctly, i.e. that the salted hash
+    Verify that creating a role with a hashed password works correctly, i.e. that the hashed password
     present in `system.roles` is the same as the one we provide.
     """
 
     with AuthSLContext(cql):
         role = "andrew"
-        # Arbitrary salted hash. Could be anything.
+        # Arbitrary hashed password. Could be anything.
```

Comment:
I guess this comment isn't accurate in Cassandra, which is why you changed the value to something that actually looks like a real salted hash in libxcrypt format.